windows-terminal-quake/Native/FocusTracker.cs

```diff
@@ -40,5 +42,49 @@ public static void FocusGained(Process process)
 				}
 			});
 		}
+
+		// Checks to see if the current window in focus should ignore hotkey
+		public static bool CheckFocusIgnoreHotKey()
+		{
+			var currentWindowHandle = User32.GetForegroundWindow();
+			var allProcesses = Process.GetProcesses();
+			
+			foreach (var window in allProcesses)
```

Comment:
You could turn this loop into a couple lines of linq, probably makes it a bit more readable.

Reply:
Agreed.
Consider it done for the next commit :)